### ISSUE TITLE
Replication slots not enabled on AWS Aurora Postgres

### ISSUE THREAD

**Issue by McDoit** [ankane/pghero#229]
Hello

First of all, i don't know if this should be a bug, idea or just help others

I ran into issues running pg_hero against AWS Aurora Postgres, with they not using replication slots and therefore throwing errors like these:
`ERROR:  Replication slots are currently not supported in Aurora`
after the following statement:
`STATEMENT:  SELECT slot_name, database, active FROM pg_replication_slots`

Luckily, with the help of the permissions guide here (https://github.com/ankane/pghero/blob/master/guides/Permissions.md) i came up with the following solution to stop the errors poluting my database logs:

```
-- query replication slots
CREATE OR REPLACE FUNCTION pghero.pg_replication_slots() RETURNS SETOF pg_replication_slots AS
$$
select
cast( null as name	) "slot_name",
cast( null as name	) "plugin",
cast( null as text	) "slot_type",
cast( null as oid	) "datoid",
cast( null as name	) "database",
cast( null as boolean	) "active",
cast( null as integer	) "active_pid",
cast( null as xid	) "xmin",
cast( null as xid	) "catalog_xmin",
cast( null as pg_lsn	) "restart_lsn",
cast( null as pg_lsn	) "confirmed_flush_lsn"
limit 0;
$$ LANGUAGE sql VOLATILE SECURITY DEFINER;

CREATE VIEW pghero.pg_replication_slots AS SELECT * FROM pghero.pg_replication_slots();
```

Is this something i sould add to the guide itself with a PR?
Other than that, thanks for an amazing product!

**Comment by ankane** [comment]
Make sure you're on the latest version of PgHero, as this was fixed in this commit: https://github.com/ankane/pghero/commit/067e25087d1cf7fb947d2131222bbe56aa4b494a

**Comment by McDoit** [comment]
Ok, sorry, I was downloading and using the docker image, and it doesn't seem to be updated then
Sadly the docker one is the only one i can use for now due to my lack of Linux knowledge

**Comment by ankane** [comment]
The Docker image is kept up-to-date as well. https://hub.docker.com/r/ankane/pghero/builds/

**Comment by McDoit** [comment]
Im running pghero on AWS ECS, and updated the task to use the :latest tag and made sure i was running 2.2
But i'm still running into the issue

```
2018-09-17 20:01:35 UTC:10.0.5.233(40246):pghero@system:[20330]:ERROR: Replication slots are currently not supported in Aurora
2018-09-17 20:01:35 UTC:10.0.5.233(40246):pghero@system:[20330]:STATEMENT: SELECT slot_name, database, active FROM pg_replication_slots
```

**Comment by ankane** [comment]
Hey @McDoit, think I may be misunderstanding the issue. Does the dashboard still work, but you're seeing errors in the database logs?

**Comment by McDoit** [comment]
> Hey @McDoit, think I may be misunderstanding the issue. Does the dashboard still work, but you're seeing errors in the database logs?

Yes, exactly, the dashboard didn't show anything, but the logs on the database was filled with several entries per minute

**Comment by ankane** [comment]
Feature support is now cached in memory on master, so you should see a lot less errors in the logs.

**Comment by ankane** [comment]
A new version of the `latest` Docker container should be available in 20 minutes so you can test it out.
